getting-started/air-gap-workload.hbs.md

```diff
@@ -19,7 +21,24 @@ To create a workload from Git through https, follow these steps:
     kubectl create secret generic custom-ca --from-file=caFile=CA_PATH -n NAMESPACE
     ```
 
-2. If you would like to pass in a custom settings.xml for Java, create a file called settings-xml.yaml similar to the following example:
+2. (Optional) If you would like to pass in a git repository login credentials along with the CA certificate, create a file called git-credentials.yaml similar to the following example:
+
+    ```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: git-ca
+      # namespace: default
+    type: Opaque
+    data:
+      username: <BASE64>
```

Comment:
should BASE64 be replaced with more appropriate name here? same in the next line too..

Reply:
I have used it from the source https://fluxcd.io/flux/components/source/gitrepositories/#https-certificate-authority:~:text=the%20URL.-,Basic%20access%20authentication,-To%20authenticate%20towards.
Please suggest what I can use instead of BASE64 if required. Thank you.